Scripts/CreatePionEfficency.sh

```diff
@@ -3,6 +3,15 @@
 # Script to create the SimEfficiency proditions content from a beam campaign.  The campaign 'configuration' field must be provided
 # stopped pion campaign is the argument
 rm $1_SimEff.txt
-mu2eGenFilterEff --out=$1_SimEff.txt --chunksize=100 
-sim.mu2e.PiminusStopsCat.$1.art sim.mu2e.PiTargetStops.$1.art sim.mu2e.PiMinusFilter.$1.art
-sed -i -e 's/dts\.mu2e\.//' -e 's/sim\.mu2e\.//' -e 's/\..*\.art//' -e 's/ IOV//' $1_SimEff.txt
+mu2eGenFilterEff --out=$1_SimEff.txt --chunksize=100 --firstLine "PionSimEfficiencies" --verbosity 3 sim.mu2e.PiBeamCat.$1.art sim.mu2e.PiTargetStops.$1.art 
+
+#for  filter version you need the eventCount of sim.mu2e.PiTargetStops.$1.art as the denominator
+PREFILTER=$(samDatasetsSummary.sh sim.mu2e.PiTargetStops.$1.art  | awk '/Triggered/ {print $2}')
+FILTER=$(samDatasetsSummary.sh sim.mu2e.PiTargetFilt.$1.art  | awk '/Triggered/ {print $2}')
+FILTEREFF=$(awk "BEGIN {printf \"%.5f\", ${FILTER}/${PREFILTER}}") 
+echo "PiTargetFilt, ${PREFILTER}, ${FILTER}, ${FILTEREFF}" >> $1_SimEff.txt
+
+TOTALWEIGHT=37468 #TODO need better way of adding to the file
```

Comment:
What is the point of these variables?  They seem to have no effect on the script.

Reply:
This is in flux, we dont know what will be needed once Michael finishes his work